config/sentry.php

```diff
@@ -5,7 +5,7 @@
     'dsn' => env('SENTRY_LARAVEL_DSN', env('SENTRY_DSN')),
 
     // capture release as git sha
-    // 'release' => trim(exec('git --git-dir ' . base_path('.git') . ' log --pretty="%h" -n1 HEAD')),
+    'release' => env('SENTRY_RELEASE', trim(exec('git --git-dir ' . base_path('.git') . ' log --pretty="%h" -n1 HEAD'))),
```

Comment:
I would be fine with this, but I am not with executing a shell command on our users machines by default, even though it looks harmless enough. Not everyone is using git or config caching which can cause issues.

```suggestion
    'release' => env('SENTRY_RELEASE'),
```

Reply:
I would also rather have this removed and go ahead with your suggestion here.